logging/logmanager/plugin_impl_log_manager.go

```diff
@@ -16,15 +16,21 @@ package logmanager
 
 import (
 	"fmt"
+	"log/syslog"
 	"net/http"
 	"os"
 	"strings"
 
+	lLogStash "github.com/bshuster-repo/logrus-logstash-hook"
```

Comment:
Better to have `snake_case` than `camelCase` here.

Reply:
I'm just curious why, because personally I prefer camel case. Also IDE advises the opposite, referencing this https://golang.org/doc/effective_go.html#mixed-caps

Reply:
Actually I would go for no alias at all.

And [Effective Go] actually advises this for packages:
> By convention, packages are given **lower case**, **single-word names**; there should be no **need for underscores or mixedCaps**
https://golang.org/doc/effective_go.html#package-names

Reply:
@pkotucek Why did you feel the need to use aliases at all? They should be used only if there are some collisions.

Reply:
I removed them.